sources/packages/forms/src/form-definitions/institutionprofilecreation.json

```diff
@@ -858,7 +860,7 @@
           "validate": {
             "required": true,
             "minLength": "",
-            "maxLength": 100,
+            "maxLength": 250,
```

Comment:
For my info, do we also not validate the max length for the legal operating name field?